<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="GPL75M3" failures="1" tests="6" name="com.techweb.helloworld.LoginFunction.Login_Function_Test" time="25.477" errors="0" timestamp="2023-05-31T11:10:00 ICT" skipped="0">
  <testcase classname="com.techweb.helloworld.LoginFunction.Login_Function_Test" name="TC_01_Register_Empty_Data" time="9.527"/>
  <system-out/>
  <testcase classname="com.techweb.helloworld.LoginFunction.Login_Function_Test" name="TC_06_Register_Invalid_Phone" time="4.718">
    <failure message="expected [Số điện thoại bắt đầu bằng: 09 - 03 - 012 - 016 - 018 - 019] but found [Số điện thoại bắt đầu bằng: 09 - 03 - 012 - 016 - 018 - 019 - 088 - 03 - 05 - 07 - 08]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Số điện thoại bắt đầu bằng: 09 - 03 - 012 - 016 - 018 - 019] but found [Số điện thoại bắt đầu bằng: 09 - 03 - 012 - 016 - 018 - 019 - 088 - 03 - 05 - 07 - 08]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.techweb.helloworld.LoginFunction.Login_Function_Test.TC_06_Register_Invalid_Phone(Login_Function_Test.java:149)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_06_Register_Invalid_Phone -->
  <system-out/>
  <testcase classname="com.techweb.helloworld.LoginFunction.Login_Function_Test" name="TC_05_Register_Incorrect_Password" time="2.765"/>
  <system-out/>
  <testcase classname="com.techweb.helloworld.LoginFunction.Login_Function_Test" name="TC_04_Register_Password_Less_Than_6_Chars" time="2.798"/>
  <system-out/>
  <testcase classname="com.techweb.helloworld.LoginFunction.Login_Function_Test" name="TC_02_Register_Invalid_Email" time="2.869"/>
  <system-out/>
  <testcase classname="com.techweb.helloworld.LoginFunction.Login_Function_Test" name="TC_03_Register_Incorrect_Email" time="2.800"/>
  <system-out/>
</testsuite> <!-- com.techweb.helloworld.LoginFunction.Login_Function_Test -->
